run_scripts/start_bag.il

```diff
@@ -1460,7 +1447,15 @@ procedure( create_layout( lib_name view_name via_tech layout_f "ttt" )
             printf("Creating %s__%s (%s)\n" lib_name cell_name view_name)
             create_layout_helper(cv tech_file inst_list rect_list via_list pin_list path_list
                                  blockage_list boundary_list polygon_list)
-
+            
```

Comment:
Removed running callbacks in create_layout_helper. Instead, after the layout has been instantiated, check whether the instance is a Pcell (1st if-else) and check whether the callback function is defined (2nd if-else). If both are true, run the callbacks. If the callback function is NOT defined, print a warning about possibly broken Pcell layouts. This seems to fix pcell layout generation (e.g. layouts are updated correctly).